modules/dxp/apps/commerce/commerce-dashboard-web/src/main/resources/META-INF/resources/forecast.jsp

```diff
@@ -51,7 +51,6 @@ if (commerceAccount != null) {
 			accountIds: <%= accountIds %>,
 			categoryIds: <%= categoryIds %>,
 			noAccountErrorMessage: Liferay.Language.get('no-account-selected'),
```

Comment:
Hi @manuelecastro,
Thanks for you fix. I see we likely have the very same issue in this line (no-account-selected). Could you update your PR and fix this too?
You might need to adjust the code in ForecastChart.es.js:48 (yet another issue I think)

Reply:
Hello @riccardo-ferrari ! I made the changes you asked. I'm just waiting for a response from Manuel Molina to see if this is a case for creating a new issue or not. :)